src/main/java/com/inqoo/trainingservice/app/models/Category.java

```diff
@@ -1,25 +1,24 @@
 package com.inqoo.trainingservice.app.models;
 
 import javax.persistence.*;
-import java.math.BigDecimal;
-import java.time.Instant;
+
 @Entity
-public class TrainingMain {
+public class Category {
```

Comment:
Please chek if it can be private/package private

Reply:
After refactor it will be private/package-private.